core/src/main/java/com/netflix/conductor/core/config/ConductorCoreConfiguration.java

```diff
@@ -47,6 +48,7 @@
 import static java.util.function.Function.identity;
 
 @Configuration(proxyBeanMethods = false)
+@EnableRetry
```

Comment:
Is this needed? I see `RetryTemplate` is created per class as needed. I think `@EnableRetry` is only needed if we use `@Retryable`?

Reply:
good catch, this was a leftover from when i was trying to use `@Retryable`